pkg/parser/common.go

```diff
@@ -38,13 +40,12 @@ var (
 // Parse wraps parser.Parse(), makes `parser` suitable for dm
 func Parse(p *parser.Parser, sql, charset, collation string) (stmt []ast.StmtNode, err error) {
 	stmts, warnings, err := p.Parse(sql, charset, collation)
-
-	for _, warning := range warnings {
-		log.Warnf("parsing sql %s:%v", sql, warning)
+	if err != nil {
+		log.L().Error("parse statement", zap.String("sql", sql), zap.Reflect("error message", err))
```

Comment:
can we use `zap.Error(err)` or `log.ShortError(err)` directly here?

Reply:
yep we can, I just want to make it same with warning messages